modules/mesh-task/config.tf

```diff
@@ -0,0 +1,62 @@
+locals {
+  # The goal here is to prune unused fields from the config.
+  # This helps reduce the size of the CONSUL_ECS_CONFIG_JSON variable,
+  # and keeps the JSON Schema in consul-ecs simple (no nullable fields).
```

Comment:
I agree. It would be awesome to relax this restriction. We probably would want to test out a large configuration E2E to see if anything breaks first though.